fitbenchmarking/parsing/tests/test_parsers.py

```diff
@@ -142,6 +142,11 @@ def test_parsers(self, file_format, test_file, expected):
         assert (test_file is not None), \
             'No test file for {}'.format(file_format)
 
+        with open(test_file) as f:
```

Comment:
Unless I've missed something obvious, maybe this could do with a comment to explain why you are doing this.